internal/installcfg/builder/builder.go

```diff
@@ -112,6 +112,18 @@ func (i *installConfigBuilder) getBasicInstallConfig(cluster *common.Cluster) (*
 		SSHKey:     cluster.SSHPublicKey,
 	}
 
+	if cfg.ControlPlane.Replicas == 2 {
```

Comment:
How will we differentiate between the arbiter case and the two node with fencing case once that comes through?